app/interactors/create_user.rb

```diff
@@ -0,0 +1,25 @@
+class CreateUser
+  include Interactor
+
+  def call
+    validate_input
+    execute
+    validate_output
+  end
+
+  private
+
+  def validate_input
+    context.fail!(error: "invalid user params") unless context.user_params
```

Comment:
@Enriikke so my intent here is "did you pass user_params to .call"? It would be an `ArgumentError` if `CreateUser.call(user_params: user_params)` doesn't get `user_params` though, right? So I guess this is really just checking if we accidentally typed something like `CreateUser.call` without any params. 

Is this how you would validate the input?